source/_docs/pruning-git-repo.md

```diff
@@ -5,35 +5,31 @@ categories: [developing]
 tags: [code, local]
 keywords: git, git commands, drupal, wordpress
 ---
-When a code repo is larger than 2GB, it increases the possibility of Git errors when committing code on Pantheon. This article walks you through pruning down a large Git repository so that your site is compatible with all functions on the platform.
+When a code repo is larger than 2GB, it increases the possibility of Git errors when committing code on Pantheon. This article walks you through pruning down a large Git repository so that your site is compatible with all functions on the platform. Please be aware that the steps described below use the BASH shell and Perl extensively. 
 
 ### Step 1: Make a local clone of the pantheon repository.
-1. Run the following command:
-```
-git clone ssh://codeserver.dev.482ae619-7334-4013-b8d2-01c2e765cc70@codeserver.dev.482ae619-7334-4013-b8d2-01c2e765cc70.drush.in:2222/~/repository.git
-```
+1. Follow [these instructions](https://pantheon.io/docs/git/#clone-your-site-codebase) to create your local clone
 2. Change directory to the repository by running: `cd repository-name`
 
 ### Step 2: Create local copies of all remote branches.
-For a branch in `$(git branch -r | grep -v HEAD | grep -v master)`, run:
 ```
-git branch --track "${BRANCH#origin/}"; done
+for BRANCH in `git branch -r | grep -v HEAD | grep -v master`; do git branch --track "${BRANCH#origin/}"; done
```

Comment:
@hengkit: What should the local branches be tracking? I'm seeing all branches track local master, such as: <code>Branch xml-sitemap set up to track <strong>local branch master</strong> by rebasing.</code> Is that the intended result? 

Suggesting we use the following instead: 
```
for BRANCH in `git branch -r | grep -vE "HEAD|master"`; do git branch --track ${BRANCH#origin/} $BRANCH; done
```
Produces: <code>Branch xml-sitemap set up to track <strong>remote branch xml-sitemap from origin</strong> by rebasing.</code> which I think is what we’d want, yeah?